apps/web/scripts/installExternalCourses.py

```diff
@@ -0,0 +1,20 @@
+import json
+import sys
+import os
+
+def main():
+    if len(sys.argv) != 2:
+        exit(f"Usage: {sys.argv[0]} PATH_TO_CONFIG_FILE")
+    filename = sys.argv[1]
+    with open(filename) as fh:
+        config = json.load(fh)
+    for course in config:
+        if course['deploy']:
+            cmd = f'''yarn installCourse "{course['url']}" {course['tdir']}'''
```

Comment:
we should probably rewrite this fully in Python then, no?

Reply:
(which is of course not to say that it needs to be done in this merge request per se)

Reply:
Well, yes, but I prefer to make changes to be smaller rather bigger.

Reply:
yes, I agree with that